subsys/mgmt/mcumgr/grp/img_mgmt/src/img_mgmt_state.c

```diff
@@ -0,0 +1,872 @@
+/*
+ * Copyright (c) 2018-2021 mcumgr authors
+ * Copyright (c) 2022-2023 Nordic Semiconductor ASA
```

Comment:
~~this has been patched in v3.2-branch though, so just update manifest and use the already patched file without duplicating it?~~ different fix